### PR TITLE
Add new bar chart

### PR DESCRIPTION
**Pull request by ndtquang**
### Card
Finish [#344](https://employmenthero.atlassian.net/jira/software/projects/ANS/boards/93?selectedIssue=ANS-344)

### Description
- Implement BarChart component
- Add CSS style for BarChart

### Screenshot
Screen Shot 2021-07-13 at 15 48 43

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -23,4 +23,49 @@ function App() {
...
+
+  return <g className="bar-group">
+    <text className="name-label" x="-6" y={yMid} alignmentBaseline="middle" >{props.d.name}</text>
+    <rect y={barPadding * 0.5} width={width} height={props.barHeight - barPadding} fill={barColour} />
+    <text className="value-label" x={width- 8} y={yMid} alignmentBaseline="middle" >{props.d.value}</text>
+  </g>
+}
+
+class BarChart extends React.Component {
+  state = {
```
Comment (phthhieu) [inline src/App.js:42]: Why it is a state instead of a constant? Since I see it is never changed.

**Review by phthhieu — APPROVED** [review]
I guess we would remove yarn.lock. (or package-json.lock)

File: src/App.js [hunk 2]
```diff
@@ -23,4 +23,49 @@ function App() {
   );
 }
 
-export default App;
+function BarGroup(props) {
```
[the remark this answers is not in the document]
Comment (haudao) [reply src/App.js:26]: You now can delete these code as you already move it to `components`

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,3 @@
 /node_modules
 .ds_store
+yarn.lock
```
Comment (haudao) [inline .gitignore:3]: This is unneccessary

**Review by haudao — APPROVED** [review]

**Review by phthhieu — APPROVED** [review]

**Pull request merged**